packages/ui/src/icons/external-icon.tsx

```diff
@@ -0,0 +1,14 @@
+import * as React from 'react';
+import { Svg } from '../svg';
+import { BoxProps } from '../box';
+
+export const ExternalIcon: React.FC<BoxProps> = props => (
```

Comment:
in cases like this, I tend to type as 

```
export const ExternalIcon = (props: BoxProps) => (
```

as, the former will not error when passed a child